BUILD.md

```diff
@@ -26,6 +26,12 @@ This is a video guide to setup Hyperledger Cactus on your local machine.
 
 [![IMAGE ALT TEXT HERE](https://img.youtube.com/vi/94xoV9Vfu14/0.jpg)](https://www.youtube.com/watch?v=94xoV9Vfu14)
 
+
+### Installing jkd8 
```

Comment:
`jdk8` a typo

Reply:
I managed to make a typo for a four character word lol, thank you for checking :)

Reply:
:smile:

Reply:
@Leeyoungone It still has the typo in there AFAICT. (Also, sorry for the slow response/review here)